backend/src/models/companyFiles.model.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  Table,
+  Column,
+  Model,
+  DataType,
+  BelongsTo,
+  ForeignKey,
+} from 'sequelize-typescript'
+import Company from './company.model'
+
+export type FileDescription =
+  | 'INE representante legal'
+  | 'Acta constitutiva'
+  | 'Curriculum'
+  | 'Directorio de instaladores certificados de CDMX'
+  | 'Padron de empresas especializadas FIDE'
+  | 'Certificaciones sistemas fotovoltaicos'
+  | 'NOM-027-ENER/SCH-2018'
+  | 'NMX-ES-004-NORMEX-2015'
+  | 'Archivos presion mayor a 294k Pa'
+  | 'Archivos presion menor a 294k Pa'
+  | 'Carta de compromiso'
+  | 'Imagen'
+  | 'Otro' // EVITAR USAR ESTE VALOR
```

Comment:
If the value is not used, delete it

Reply:
no, denisse suggested to leave it, please chill bro

Reply:
Lo veo bien, LGTM :)

Reply:
Denisse confirmed she did not suggested that OTHER